eventmesh-webhook/eventmesh-webhook-api/src/main/java/org/apache/eventmesh/webhook/api/WebHookOperationConstant.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.eventmesh.webhook.api;
+
+import java.io.File;
+
+/**
+ * Webhook常量类
+ */
+public class WebHookOperationConstant {
+
+    public static final String FILE_SEPARATOR = File.separator;
+
+    public static final String FILE_EXTENSION = ".json";
+
+    public static final String GROUP_PREFIX = "webhook_" ;
+
+    public static final String DATA_ID_EXTENSION = ".json";
+
+    public static final String MANUFACTURERS_DATA_ID = "manufacturers" + DATA_ID_EXTENSION;
+
+    public static final Integer TIMEOUT_MS = 3*1000;
```

Comment:
Use int here will cause extra implicit conversion.
```suggestion
    public static final long TIMEOUT_MS = 3 * 1000L;
```